.github/workflows/ci.yml

```diff
@@ -56,7 +56,15 @@ jobs:
         with:
           arguments: check
 
-      # TODO eventually also run instrumentation/UI tests?
+      - name: Run instrumentation tests
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 33
+          arch: x86_64
+          disable-animations: true
+          disk-size: 2000M
+          heap-size: 600M
+          script: ./gradlew connectedDebugAndroidTest --stacktrace
```

Comment:
I'm not sure this will pass without any tests to run. Why not just fold it into your current PR?

Reply:
I had planned to do so. I thought your ask was to get it in place before merging #43. If you don't care I'll close this and merge with #43

Reply:
ah oops, sorry meant do it that PR but before you merge 😅